scripts/data_transfer/copy_test_data_for_reports.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copy a sample-QC metadata TSV file generated by the joint-calling pipeline into the test bucket.
+The idea is that for interactive exploration of QC summary metrics and design of plots, it's safe
+to allow using the full set of samples, because the metadata doesn't contain any actual genomics
+variantion data. 
+
+Note that the test bucket also contains gs://cpg-tob-wgs-test/joint-calling/v2/meta.tsv, which
+is symmetrical to gs://cpg-tob-wgs-main/joint-calling/v2/meta.tsv and limited to samples in the
+test subset.
+"""
+
+import os
+import subprocess
+import click
+
+output = os.getenv('OUTPUT')
+assert output and output.startswith('gs://cpg-tob-wgs-test/')
+
+
+@click.command()
+@click.option('--version', help='Joint-calling run version', required=True)
+def main(version):
+    """Entry point"""
+
+    subprocess.run(
+        [
+            'gsutil',
+            'cp',
+            f'gs://cpg-tob-wgs-main/joint-calling/{version}/meta.tsv',
+            f'gs://cpg-tob-wgs-main-metadata/joint-calling/{version}/meta.tsv',
```

Comment:
Did you mean to use `output` here?

Reply:
But since we don't want the destination to be configurable, why use a variable here?

Reply:
Well, we also copy a metadata file to `output` in the analysis-runner that describes that particular run. If you're not actually writing to that location, these destinations can be out of sync.